.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "schema"]
```

Comment:
I should check, but I don't think people have to do anything different for things to work. But I should probably add a note about how to update the submodule.